src/components/Gallery/Gallery.tsx

```diff
@@ -49,6 +50,7 @@ const getSizes = (isLead: boolean, isPortrait: boolean) => {
 
 export const GalleryMedia = ({
   alt,
+  canDownload,
```

Comment:
could also add a default setting to this prop declaration e.g. `canDownload = false,` so no download available unless explicitly true